test-integration/scripts/01-configure-azure.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -e
+source $(dirname $0)/00-init-env.sh
+
+if [[ "$JHI_PROFILE" == "" ]]; then
+    echo "##vso[task.setvariable variable=JHI_PROFILE]dev,webapp"
```

Comment:
Is "webapp" still needed?

Reply:
```suggestion
    echo "##vso[task.setvariable variable=JHI_PROFILE]dev"
```
Since we are switching to npm scripts, no.
Old ci scripts still needs it.